component/notification/src/main/java/org/exoplatform/social/notification/plugin/ActivityCommentPlugin.java

```diff
@@ -87,7 +86,12 @@ public boolean isValid(NotificationContext ctx) {
     ExoSocialActivity comment = ctx.value(SocialNotificationUtils.ACTIVITY);
     ExoSocialActivity activity = Utils.getActivityManager().getParentActivity(comment);
 
-    if(isSubComment && comment.getParentCommentId() == null) {
+    if (StringUtils.equals(activity.getType(), "news") && !Utils.isActivityNotificationsEnabled("news") ||
+            StringUtils.equals(activity.getType(), "shared_news") && !Utils.isActivityNotificationsEnabled("shared_news")) {
+      return false;
```

Comment:
No need to hardcode news and shared_news, just use the new API :+1: 
```suggestion
    if (!Utils.isActivityNotificationsEnabled(activity.getType())) {
      return false;
```

Reply:
Yes you are right (y)